docs/antrea-network-policy.md

```diff
@@ -644,8 +644,8 @@ which the rule is applied. These log files can then be retrieved for further
 analysis. By default, rules are not logged. The example policy logs all
 traffic that matches the "DropToThirdParty" egress rule, while the rule
 "AllowFromFrontend" is not logged. Specifically for drop and reject rules,
-deduplication is applied to simplify multiple logs. Duplication buffer length is set as 1 second.
-The rules are logged in the following format:
+deduplication is applied to simplify multiple logs. Duplication buffer length
```

Comment:
What does "simplify" mean? Should it be "to remove duplicated logs" or "to reduce duplicated logs"?

Reply:
I feel readers might not understand what does "Duplication buffer length is set as 1 second" mean. But anyway it might be fine to keep it. How about changing to ", and duplication buffer length is set to 1 second"?

Reply:
Done.